spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/TracingProperties.java

```diff
@@ -33,6 +35,17 @@ public class TracingProperties {
      */
     private boolean showEventSourcingHandlers = false;
 
+    /**
+     * Whether to nest spans of subsequent in the same trace. This setting is disabled by default.
```

Comment:
Wouldn't hurt to specify in the documentation this setting effects events only. And also the field, actually.